.circleci/config.yml

```diff
@@ -13,10 +13,11 @@ jobs:
     steps:
       - checkout
       - setup_remote_docker:
-          version: 20.10.11
+          version: "20.10.11"
       - run:
           name: "Build Docker Images"
           command: |
+            ./check-versions.sh
```

Comment:
I added a commit to move this line around. It was in the deploy step which won't run for PRs.